content/project-news/june-2020-meeting-minutes.md

```diff
@@ -0,0 +1,91 @@
+
+* Point 1: Achievements and Future Outlook
+* Point 2: GSoC 2020 Status
+* Point 3: Channel moderation
+* Point 4: 0.4.14 release status
+* Point 5: Unifying our contribution guidelines and handling Pull Requests
+
+## Point 1: Achievements and Future Outlook
+
+**Amine Khaldi** worked on the built-in PCH CMake, the flex/bison PR and some other minor tasks.
+
+**Can Taşan** continued to test Microsoft Office 2003 on ReactOS and caught two regressions.
+Meanwhile, he opened a Turkish ReactOS Twitter account to promote ReactOS.
+
+**Colin Finck** released RosBE 2.2.1 and he's now checking out libc++ for ReactOS with Dominik.
+He's also keeping our infrastructure alive and mentoring a GSoC student.
+
+**Mark Jansen** has been doing some improvements / fixes in the shell, and fixed some headers where structs were wrong.
+He also worked with Hervé on the kernel shim engine.
+It's not completed yet, but the first part is available in a PR.
+
+**Stanislav Motylkov** joined in late, but still posted his status report.
+He has been helping Victor with PnP manager testing and he has been helping genedisean with the PC-98 port when possible.
+He has also been testing the Xbox port from time to time, and he has been spreading the word about ReactOS in some places where we can hopefully attract new developers.
+
+**Victor Perevertkin** worked on build systems and transitioning to GitHub Actions CI.
+At the moment he's mostly busy with kernel work: making the PnP Storage Stack work.
+He decided to do this through implementing PnP support in scsiport.sys so it can wrap a legacy uniata driver, and attach it properly to the PnP storage stack.
+For this to work properly, IO/PnP kernel changes are required, which is what he has been busy with the past two weeks.
+
+## Point 2: GSoC 2020 Status
+The mentors reported on how their students were doing.
+
+## Point 3: Channel moderation
+Daniel Reimer talked about problems which makes moderation harder than it should be.
+Victor Perevertkin suggested the topic about hiring a web developer once again, but noted that there's currently no one willing to manage this.
+Colin said he would outline his plan in a JIRA issue in the next few days, which may make it easier to find a skilled person to work on that.
+
+## Point 4: 0.4.14 release status
+Joachim was not available to report about the release itself, so Bișoc George was asked about the status on the press release.
+
+He reported that it's almost ready.
+Foxlet provided all the press media kit screenshots with his design.
+What's left is to bring the press document to Zachary for English grammar corrections and such.
+
+## Point 5: Unifying our contribution guidelines and handling Pull Requests
+For the last agenda point, Victor had already created a document outlining his thoughts.
+He gave a quick recap in the meeting channel about the two basic goals he wanted to achieve:
+
+* Reduce the decision time: reject PR or not, and reduce overall review time of course
```

Comment:
`of course` feels out of place here,
do you intend it to 'review the time of the course' or 'of course we want to reduce the overall review time' ?
For the former, 'of the course' might be better, for the latter, I'd drop 'of course'

Reply:
Yes, I agree. Those lines were taken as-is from Victor's recap. It sounds like he meant the latter, so I dropped it. 👍

Reply:
Yeah, I guess I missed an article here